health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/index.js

```
@@ -25,9 +33,28 @@
  */
 const CampaignBreadCrumb = ({ location, defaultPath }) => {
   const { t } = useTranslation();
-  
+
   const search = useLocation().search;
+  const queryParams = new URLSearchParams(search);
+  const history = useHistory();
+  const [campaignNumber, setCampaignNumber] = useState(queryParams.get("campaignNumber"));
+  const tenantId = Digit.ULBService.getCurrentTenantId();
   const pathVar = location.pathname.replace(defaultPath + "/", "").split("?")?.[0];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)